go/libraries/doltcore/sqle/enginetest/dolt_transaction_commit_test.go

```diff
@@ -0,0 +1,270 @@
+package enginetest
+
+import (
+	"context"
+	"testing"
+
+	"github.com/dolthub/go-mysql-server/enginetest"
+	"github.com/dolthub/go-mysql-server/sql"
+	"github.com/stretchr/testify/require"
+
+	"github.com/dolthub/dolt/go/libraries/doltcore/doltdb"
+	"github.com/dolthub/dolt/go/libraries/doltcore/sqle"
+)
+
+func TestDoltTransactionCommitOneClient(t *testing.T) {
```

Comment:
These tests should also check that the db_head variable moves after a commit

This is subtle and probably doesn't work correctly right now. The variable has to move for the client that commits, but not for the other client until they begin a new tx.

And at that point, they need to see the new head value

Reply:
Ohh yeah I’ll add that